finder-app/Makefile

```diff
@@ -0,0 +1,8 @@
+CROSS_COMPILE=
+CC=gcc
+
+all:
+	$(CROSS_COMPILE)$(CC) writer.c -Wall -Werror -o writer
```

Comment:
@Ruthvik-Chanda see class notes from today for the recommended way to handle this.